# Research scientist, NCBI/NLM/NIH
mailto: sergey.shmakov at gmail
#### Summary: 
I am a research fellow with nine years of research and six years of software development experience. My research field is computational biology, where I develop and apply computational methods to analyze bacterial and viral genomes. I have discovered and characterized 14 novel CRISPR-Cas systems, participated in the classification of CRISPR-Cas systems, identified crRNA-like regulatory elements in model organisms and created a protocol to search for functionally associated genes. My research was published in high profile journals like Nature, Cell and Science. 
* * *

#### Research areas:
CRISPR-Cas and Bacterial Defense Systems; Genome Mining; Comparative and Evolutionary Genomics; Bioinformatics; Machine Learning;
* * *

#### Skills:
**Biology:** Bacterial Genomes, Mobile Genetic Elements: CRISPR-Cas, Defense Systems, Transposons, Viruses;<br>
**Bioinformatics:** Sequence analys: BLAST, HHSearch, MMSeqs, AlphaFold2, DALI, FoldSeek; Phylogeny: FastTree, IQTree; Genomic Databaseses assembly and maintaining;<br>   
**Soft. Dev.:** Python, R, Bash, previously used: C/C++, C#;<br>OOP, Functional Programming, Unit tests, Massive data processing, parallel computing, HPC;<br>
**ML tools:** PyTorch, PyTorch Geometric;<br>
**OS platform:** Linux, Windows OS;
* * *

#### Experience:
**10/2017 – Present:** Evolutionary Genomics Research group, NCBI/NLM/NIH, Bethesda, MD, USA<br>Postdoctoral Research under Dr. Eugene Koonin.
  - Developed computational methods and published [protocol](https://pubmed.ncbi.nlm.nih.gov/31520072/) to identify functionally linked genes in prokaryotes.<br>
  - Discovered 14 novel CRISPR-Cas systems using the developed bioinfomratic methods. I found and characterized completely new [Type VI](https://pubmed.ncbi.nlm.nih.gov/26593719/) and other [class 2 CRISPR-Cas systems](https://pubmed.ncbi.nlm.nih.gov/31857715/).<br>
  - Identified new type of [crRNA-based regulatory elements](https://pubmed.ncbi.nlm.nih.gov/37283088/) including novel regulators in model organisms.<br>
  - Adopting Graph Neural Network-based methods for targeted search for novel prokaryotic systems (unpublished).

**10/2017 – 10/2018:** Skolkovo Institute of Science and Technology, Moscow, Russia<br>Research Fellow under Dr. Konstantin Severinov<br>
**3/2015 – 10/2017:** Evolutionary Genomics Research group, NCBI/NLM/NIH, Bethesda, MD, USA<br>Graduate Research under Dr. Eugene Koonin and Dr. Konstantin Severinov<br>
**2/2008 – 07/2014:** Microsoft (Russia), Moscow, Russia<br>Software Developer Engineer (SDE), Microsoft Dynamics department<br>
  - Was involved in for: Feature development; Test automations; Architecture solutions; Internal and vendor code review, product legacy code refactoring; Agile development process (SCRUM/KANBAN);<br>
  - Main projects: Developed backbone for Microsoft Dynamics NAV Rapid Start, excel reporting engine;<br>
  - Technologies used: Microsoft Dynamics NAV; C/AL; C#; Microsoft Azure; 
  
* * *

#### Education:
**8/2014 – 10/2017:** Skolkovo Institute of Science and Technology, Moscow, Russia<br>Ph.D. in Computational Biology and Bioinformatics<br>
**8/2012 – 8/2014:** Moscow Bioinformatics School, Moscow, Russia<br>Two-year bioinformatic school led by Mikhail Gelfand with focus on bioinformatics, statistics, mathematics, and molecular biology<br>
**9/2006 – 8/2008:** Bauman Moscow State Technical University, Moscow, Russia<br>M.Sc. in Computer science<br>
**9/2002 – 8/2006:** Bauman Moscow State Technical University, Moscow, Russia<br>B.Sc. in Computer science
* * *

#### Publications:
Published 31 papers that have more than 8000 citations and h-index of 25
Complete list available at [Google Scholar](https://scholar.google.com/citations?hl=en&user=X6lCOCAAAAAJ&view_op=list_works)

##### Selected publications:
*   Shmakov SA, et al. **Widespread CRISPR-derived RNA regulatory elements in CRISPR-Cas systems.** Nucleic Acids Res. 2023. doi: 10.1093/nar/gkad495<br>
*   Shmakov SA, Utkina I, et. al. **CRISPR Arrays Away from cas Genes.** CRISPR J. 2020. doi: 10.1089/crispr.2020.0062<br>
*   Shmakov SA, et al. **Mapping CRISPR spaceromes reveals vast host-specific viromes of prokaryotes.** Commun Biol. 2020. doi:10.1038/s42003-020-1014-1<br>
*   Shmakov SA, et al. **Systematic prediction of functionally linked genes in bacterial and archaeal genomes.** Nat Protoc. 2019. doi: 10.1038/s41596-019-0211-1<br>
*   Shmakov SA, et al. **Systematic prediction of genes functionally linked to CRISPR-Cas systems by gene neighborhood analysis.** Proc Natl Acad Sci USA. 2018. doi: 10.1073/pnas.1803440115<br>
*   Shmakov SA, et al. **The CRISPR Spacer Space Is Dominated by Sequences from Species-Specific Mobilomes.** MBio. 2017. doi: 10.1128/mBio.01397-17<br>
*   Shmakov S, et al. **Diversity and evolution of class 2 CRISPR-Cas systems.** Nat Rev Microbiol. 2017. doi:10.1038/nrmicro.2016.184<br>
*   Shmakov S, et al. **Discovery and Functional Characterization of Diverse Class 2 CRISPR-Cas Systems.** Mol Cell. 2015. doi: 10.1016/j.molcel.2015.10.008<br>

#### Patents:
“Novel crispr enzymes and systems”:<br>
U.S. Patent Application No. 2016/038238 (published Mar. 2016-12-22)<br>
U.S. Patent Application No. 2016/038258 (published Mar. 2016-12-22)
* * *

#### Scientific presentations:
*   6th International Conference on CRISPR Technologies, Boston, MA, USA 2023<br>Talk: CRISPR-like RNAs regulate expression of cas genes<br style="line-height: 10px" />
*   CRISPR 2022, Cambridge, MA, USA 2022<br>Poster: Repeat-like RNAs in CRISPR-Cas loci<br style="line-height: 10px" />
*   CRISPR 2019, Quebec City, Quebec, Canada 2018<br>Poster: Mapping the CRISPR-Cas spacer space onto prokaryotic virome<br style="line-height: 10px" />
*   NIH Research Festival 2018, Bethesda, MD, USA 2018<br>Oral presentation: Computational approaches to discover novel CRISPR-Cas systems<br style="line-height: 10px" />
*   CRISPR 2018, Vilnius, Lithuania 2018<br>Poster: Systematic prediction of genes functionally linked to CRISPR-Cas systems by gene neighborhood analysis<br style="line-height: 10px" />
*   CRISPR 2017, Big Sky, MT, USA 2017<br>Poster: The CRISPR Spacerome<br style="line-height: 10px" />
*   CRISPR 2016, Rehovot, Israel 2016<br>Poster: CRISPR Effector Discovery Pipeline<br style="line-height: 10px" />
*   GeneHack, Hackathon, Moscow, Russia 2016<br>Oral presentation: CRISPR-Cas Class II search<br style="line-height: 10px" />
*   Scientific Computational Biology branch, Retreat, Bethesda, MD, USA 2016<br>Oral presentation: Novel CRISPR-Cas systems<br style="line-height: 10px" />
*   Skoltech MIT Conference, Moscow, Russia 2016<br>Oral presentation: Novel CRISPR-Cas systems<br>

#### Awards:
*   Moscow Government Prize 2019<br>Discovery and characterization of novel prokaryotic defense systems<br style="line-height: 10px" />
*   Research grant from Russian Fund for Fundamental Research #18-34-00012 2017-2018<br>Discovery and characterization of novel CRISPR-Cas systems<br style="line-height: 10px" />
*   Philip Morris Systems Biology Fellowship Program 2018<br>Comprehensive characterization of CRISPR-linked genes<br style="line-height: 10px" />
*   NIH Director’s Award 2016<br>Discovery of novel class 2 CRISPR-Cas systems<br style="line-height: 10px" />
*   Skoltech President Stipend Award 2016<br>Discovery of novel class 2 CRISPR-Cas systems<br>

#### Teaching and Mentorship:
*   Koonin Laboratory, Bethesda, MD, USA 2018<br>Co-mentored master student, Irina Utkina<br>
*   Koonin Laboratory, Bethesda, MD, USA 2016<br>Participated in 9-week Scientist Teaching Science Pedagogy course<br>
*   GeneHack, Hackathon, Moscow, Russia 2016<br>Led two tracks related to CRISPR-Cas Class II search, mentored participating teams<br>

#### Professional Services:
Served as reviewer for: Nature Scientific Reports, Oxford Academic Database,
Frontiers in Microbiology, PeerJ The Journal of Life and Environmental Sciences, MDPI Biology, KWF 
Kankerbestrijding – Dutch Cancer Society
* * *

#### Languages:
Russian - Native<br>English - Fluent